### DISCUSSION TITLE
Versionamento

### DISCUSSION THREAD

**Discussion by LuizHAP** [diego3g/rsxp-2023#34] [General]
Indo em linha que o aplicativo terá uma versão para Apple Store (acredito eu). Tenho algumas perguntas:

- Devemos se preocupar com a publicação? 

Podemos criar uma release inicial, e já enviar para aprovação da Apple Store (pq até aonde eu sei demora um pouco pra aprovar)
Quando podemos começar a pensar em já enviar?

- Versionamento do código

Teremos changelog? Como será a questão de regras do versionamento? Minha contribuição seria utilizar o [SEMVer](https://semver.org/lang/pt-BR/) juntamente com o Commitzen (http://commitizen.github.io/cz-cli/) além de adicionar as alterações no CHANGELOG.md.

O que acham?

**Comment by CassianoJunior** [comment]
Também sugeri isso pro versionamento. Dá uma olhada na #26

**Comment by diego3g** [comment]
Boa @LuizHAP, acho que podemos enviar assim que tivermos a feature de resgatar ingresso, se não, eles nem aprovam o app.

Sobre o padrão de commit, recomendo seguirmos a discussão na thread #26.
